### ISSUE TITLE
OWL: support nominals

### ISSUE THREAD

**Issue by mkroetzsch** [knowsys/vlog4j#20]
Support for owl:oneOf is currently not implemented yet and this is not correctly reported with an exception. This should be fixed by supporting all admissible nominals in subclasses and superclasses. The architecture already has most of the functionality required, but variable substitutions should be added (rather than just relying on auxiliary facts as possible in subclasses.

**Comment by irina-dragoste** [comment]
Axiom `SubClassOf(<A> ObjectOneOf(<a>))`
is incorrectly transformed into the rule` http://www.w3.org/2002/07/owl#Nothing(?X) :- A(?X)`

**Comment by irina-dragoste** [comment]
Axioms `SubClassOf(ObjectOneOf(<a> <b>) <A>)` and `SubClassOf(ObjectOneOf(<a>) <A>)` are incorrectly transformed into the program 
```
rules: [A(?X) :- http://www.w3.org/2002/07/owl#Thing(?X)]
facts: []
```

**Comment by irina-dragoste** [comment]
Partially fixed by #132: we support nominals in axioms of type subClassOf , with a ObjectOneOf class expression subtype. (ex: {a,b,c} subClassOf anyClassExpression). For any other nominal occurrences, we throw an exception.

The remaining fixes are in issue #142.
